Task_2.1.2/User.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Figures
+{
+    public class User
+    {
+        public string Name { get; set; }
+        public List<Figure> _figures;
+
+        public User(string name)
+        {
+            Name = name;
+            _figures = new List<Figure>();
+           
+        }
+
+        public static implicit operator User(string name)
+        {
+            return new User(name) { Name = name, _figures = new List<Figure>() };
```

Comment:
```suggestion
            return new User(name);
```
Все то, что ты написала в инициализаторе, и так написано в конструкторе